lib/jira/commands/install.rb

```diff
@@ -15,35 +15,52 @@ class Install < Base
 
       def run
         io.say('Please enter your JIRA information.')
-        inifile[:global] = params
+        inifile[:global] = base_params
+        inifile.write # Do this now because cookie authentication uses api calls
+
+        inifile.delete_section("cookie") if inifile.has_section?("cookie")
+        case authentication
+        when "basic"
+          inifile[:global][:password] = password
+        when "token"
+          inifile[:global][:token] = token
+        when "cookie"
```

Comment:
Do we know how long the session will last before JIRA expires the session? We should store the TTL
We don't want to have the user choose cookie auth and then 2 weeks later start receiving random error messages because the session expired